src/modules/entities/modules/reports/services/reducers/entities/list.js

```diff
@@ -5,7 +5,7 @@ const initialState = [];
 export default function listReportsReducer(state = initialState, action) {
     switch (action.type) {
         case types.FETCH_REPORTS_SUCCESS:
-            return [...action.payload];
+            return [...action.meta.id];
```

Comment:
what is this? this should return the list of reports and not the id from meta

Reply:
this is not a good pattern to use the action creator with the detail request

we will have to refactor it and all `fetchReports` actions rewrite without redux-utils ... like this e.g. - https://gitlab.ack.ee/Web/flash-news-cms/-/blob/development/src/modules/entities/modules/data-channels/services/actions/fetchDataChannels.js